README.md

```diff
@@ -6,8 +6,13 @@
 # About
 Add a simple glossary panel to your site to help your users understand jargon-y terms. As seen on [betaFEC](https://beta.fec.gov) and [EITI](https://useiti.doi.gov/).
 
+For example:
+* https://beta.fec.gov/data/ - click on the book icon in the header
+* https://useiti.doi.gov/how-it-works/production/ - click on "fossil fuel"
```

Comment:
FWIW, eiti doesn't actually seem to use this library--looks like they've vendorized and modified an older version that's included directly in their source. See https://github.com/18F/doi-extractives-data/blob/dev/js/components/glossary.js.